tests/queries/0_stateless/02677_hash_functions_on_null.reference

```diff
@@ -0,0 +1,4 @@
+42
```

Comment:
maybe it makes sense to make it equal to `xxHashN()` (with no args)

Reply:
Yes, done.